docs/extensions/consensus/_category_.json

```diff
@@ -0,0 +1,5 @@
+{
+    "label": "Coordinated Upgrades",
+    "position": 7,
```

Comment:
I think we should have coordinate upgrades be above `utilities` in the sidebar. This would be position 6, and utilities would be position 7